Lists installed hardsuit modules when examining

Kasuobes opened the pull request:
Similar to how examining mechs will list all the installed equipment, RIGs will now list all installed modules if the panel is open, while the RIG is being held or worn.

Primarily a QOL improvement for ERT so players won't have to wear the RIG, activate it, check the interface for the list of modules, then deactivate and remove the RIG to install extra modules (or shoving in modules randomly only to realize you can't install two laser cannons). 

Also benefits merc players so they won't buy duplicate modules with their telecrystals and beg for refunds in ahelp.

Kasuobes committed:
Lists installed hardsuit modules when examining

Similar to how examining mechs will list all the installed equipment, RIGs will now list all installed modules if the panel is open, while the RIG is being held or worn.

PsiOmegaDelta commented:
:+1:

PsiOmegaDelta commented on code/modules/clothing/spacesuits/rig/rig.dm:
```diff
@@ -96,9 +96,16 @@
+		if(open && installed_modules.len)
+			usr << "It's equipped with:"
+			for(var/obj/item/rig_module/E in installed_modules)
+				usr << "[E]"
```
`usr << E` would have the same effect.
If you want to avoid "The Module Name", use `usr << E.name`.

GinjaNinja32 replied:
alternatively, use `"It's equipped with: [english_list(installed_modules)]"`.

As currently implemented, `english_list` doesn't quite handle instances right (you end up with "The one, The two and the three"), but I plan to poke it to make it optional between "one, two and three", "the one, the two and the three", and "The one, the two and the three".

GinjaNinja32 replied:
i have done this, though without the optiony bits. #13852. `"It's equipped with [english_list(installed_modules)]"`. Don't even need to check the length, just feed it the list and it'll say "it's equipped with nothing" if there's nothing :P

Kasuobes committed:
Squished list of isntalled modules if any

It looks prettier! :D Also won't spam the chat log as much if someone has a dozen modules installed.

PsiOmegaDelta commented on code/modules/clothing/spacesuits/rig/rig.dm:
```diff
@@ -96,9 +96,14 @@
 
+		if(open)
+			for(var/obj/item/rig_module/E in installed_modules)
+				usr << "It's equipped with [english_list(installed_modules)]."
```
With this variant you'll print all installed modules once for each installed module.
You don't need the loop.

Kasuobes replied:
Ah, I see.

GinjaNinja32 commented:
:+1:
